lib/fenrir_view/engine.rb

```diff
@@ -22,6 +22,12 @@ class Engine < ::Rails::Engine
       end
     end
 
+    initializer "fenrir_view.property_validation" do |app|
+      FenrirView.configure do |c|
+        c.property_validation ||= !Rails.env.production?
```

Comment:
Was Fenrir always/already Rails-specific?

Reply:
This entire file is ignored if Rails is not defined, but that said I've never tested this or the gem I forked the base code from if it worked without Rails.

The purpose of this thing is mainly to support a component based front-end in rails, with React or similar there are plenty of good plugins to use that renders styleguides and handles most of these other features :)